examples/src/main/java/io/dapr/examples/workflows/README.md

```diff
@@ -681,6 +682,155 @@ Key Points:
 4. Each activity simulates work with a short delay for demonstration purposes
 
 
+### Cross-App Pattern
+
+The cross-app pattern allows workflows to call activities that are hosted in different Dapr applications. This is useful for microservices architectures allowing multiple applications to host activities that can be orchestrated by Dapr Workflows.
+
+The `CrossAppWorkflow` class defines the workflow. It demonstrates calling activities in different apps using the `appId` parameter in `WorkflowTaskOptions`. See the code snippet below:
+```java
+public class CrossAppWorkflow implements Workflow {
+  @Override
+  public WorkflowStub create() {
+      return ctx -> {
```

Comment:
We can get a reference to logger and use it instead of repeating `ctx.getLogger()`. Also we can use placeholders in `logger.info()` calls instead of concatenating strings.